pkg/bicep/testdata/template.bicep

```diff
@@ -49,5 +49,6 @@ param testSecureString string
 @secure()
 param testSecureObject object
 
-resource whatever 'foobar' = {
-}
\ No newline at end of file
+resource whatever 'Microsoft.Network/virtualNetworks@2023-11-01' = {
```

Comment:
bicep linter gave error about invalid resource type, so populated it with a real one and gave it a name to satisfy linter